src/llm_interface.jl

```diff
@@ -148,6 +148,17 @@ Requires two environment variables to be set:
 """
 struct DatabricksOpenAISchema <: AbstractOpenAISchema end
 
+"""
+   AzureOpenAISchema
+
+AzureOpenAISchema() allows user to call Azure OpenAI API. [API Reference](https://learn.microsoft.com/en-us/azure/ai-services/openai/reference)
+
+Requires two environment variables to be set:
+- `AZURE_OPENAI_API_KEY`: Azure token
+- `AZURE_OPENAI_HOST`: Address of the Azure resource (`"https://<resource>.openai.azure.com/"`)
```

Comment:
```suggestion
- `AZURE_OPENAI_HOST`: Address of the Azure resource (`"https://<resource>.openai.azure.com"`; note the absence of the forward slash at the end of the URL!)
```

This is a bad example! The extra slash in the end would break the URL composition...